vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -17,6 +17,7 @@
 from vllm.config import VllmConfig
 from vllm.distributed.kv_transfer.kv_connector.v1.base import (
     KVConnectorBase_V1, KVConnectorMetadata, KVConnectorRole)
+from vllm.distributed.parallel_state import get_tensor_model_parallel_rank
```

Comment:
in follow up: do this by passing the rank to the Connector directly rather than grabbing it here.